tasks/section5.yml

```diff
@@ -471,15 +621,27 @@
       - rule_5.2.16
 
 - name: "SCORED | 5.2.16 | PATCH | Ensure SSH warning banner is configured"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^Banner'
+      line: 'Banner /etc/issue'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.16
 
 - name: "SCORED | 5.3.1 | AUDIT | Ensure password creation requirements are configured"
-  command: /bin/true
+  shell: '{{ item }}'
+  with_items:
+    - grep pam_pwquality.so /etc/pam.d/password-auth
+    - grep pam_pwquality.so /etc/pam.d/system-auth
+    - grep ^minlen /etc/security/pwquality.conf
+    - grep ^dcredit /etc/security/pwquality.conf
+    - grep ^lcredit /etc/security/pwquality.conf
+    - grep ^ocredit /etc/security/pwquality.conf
+    - grep ^ucredit /etc/security/pwquality.conf
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*